eng/testing/performance/create-provisioning-profile.sh

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env bash
```

Comment:
Is this copied from another location? If so can we put a comment so that we know where this comes from if we need to update this file.

Reply:
@akoeplinger is this something we would have to worry about needing updates from someplace else?

Reply:
no, this is a script I wrote myself.